common/service/sensor/pdr.h

```diff
@@ -0,0 +1,87 @@
+#ifndef PDR_H
+#define PDR_H
+
+#include <stdbool.h>
+#include <stdint.h>
+
+extern uint32_t pdr_count;
```

Comment:
Any specific reason you want to expose these extern vars rather than using an API? I already see an API with name plat_get_pdr_size, better to expose that, rather defining extern vars being used outside which impacts code readability and maintainability.
Same goes for other extern vars defined in file.

Reply:
Since some vars need to be initialized in platform layer, the vars are declared as extern variable for platform code to access.

Reply:
My suggestion is to define get , set APIs rather than externs. With extern vars the code browsing is hard.

Reply:
Done.